Store/StorePageFactory.php

```diff
@@ -24,7 +24,7 @@ public function __construct(SiteApplication $app)
 	// }}}
 	// {{{ public function resolvePage()
 
-	public function resolvePage($source, SiteLayout $layout = null)
+	public function resolvePage($source, SiteLayout $layout = null): SiteAbstractPage
```

Comment:
Should this be `(string, ?SiteLayout) => SiteAbstractPage`?

Reply:
Probably.  I wasn't trying to be comprehensive in changing method signatures ... just where phpstan complained that they didn't match the method they were overriding.

I see that Site has `?SiteLayout` though, so ✅